src/domain/entities/Settings.cpp

```diff
@@ -65,6 +69,26 @@ void Settings::setDeviceName(String deviceName)
     this->_deviceName = deviceName;
 }
 
+String Settings::getWifiSSID()
+{
+    return this->_wifiSSID;
+}
+
+void Settings::setWifiSSID(String wifiSSID)
+{
+    this->_wifiSSID = wifiSSID;
+}
+
+String Settings::getWifiPassword()
+{
+    return this->_wifiPassword;
+}
+
+void Settings::setWifiPassword(String wifiPassword)
+{
+    this->_wifiPassword = wifiPassword;
+}
```

Comment:
Please rename `wifiSSID` to `webuiWifiSSID` and `wifiPassword` to `webuiWifiPassword`.